docs/workshops/advanced/bootstrap.md

```diff
@@ -90,6 +90,16 @@ COMPOSE_PROFILES=lso docker compose up
 This will also enable LSO, so network devices will be actually configured. If you don't want this, just use:
 
 ### Local environment
+
+The environment requires several ports to be free.
+Use `ss` to check if any are in use—no output means they're available:
+
+```
+ss -tulnp|grep -E ':80|:3000|:4000|:5432|:5678|:8000|:8001|:8080'
+```
```

Comment:
What is `ss`?  It doesn't seem to be a default tool on macos, ubuntu or alpine.

It is a good idea to inform people to check that the ports are free, but the actual numbers written out here are bound to go stale over time. What do you think about rephrasing this to something along the lines off "Make sure you aren't running any software on the ports required by [docker-compose.yml](link/to/docker-compose.yml)" ?